app/views/healthcheck.py

```diff
@@ -0,0 +1,47 @@
+from app.models import db
+from celery.task.control import inspect
+from errno import errorcode
+from subprocess import check_output
+
+
+def health_check_celery():
+    """
+    Check health status of celery and redis broker
+    :return:
+    """
+    try:
+        d = inspect().stats()
+        if not d:
+            return False, 'No running Celery workers were found.'
+    except IOError as e:
+        msg = "Error connecting to the backend: " + str(e)
+        if len(e.args) > 0 and errorcode.get(e.args[0]) == 'ECONNREFUSED':
+            msg += ' Check that the Redis server is running.'
+        return False, msg
+    except ImportError as e:
+        return False, str(e)
+    return True, 'celery ok'
+
+
+def health_check_db():
+    """
+    Check health status of db
+    :return:
+    """
+    try:
+        db.session.execute('SELECT 1')
+        return True, 'database ok'
+    except:
+        return False, 'Error connecting to database'
+
+
+def health_check_migrations():
```

Comment:
This isn't of any use here. If there were multiple heads, the server wouldn't start and kubernetes would roll back to the previous version that worked.

What we need is the check that ensures the models are inline with the migration. A simple select query from each model would tell us that.